cmd/utils/flags.go

```diff
@@ -234,8 +234,12 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
-	WhitelistFlag = cli.StringFlag{
+	DeprecatedAllowListFlag = cli.StringFlag{
 		Name:  "whitelist",
+		Usage: "[DEPRECATED: you shall replace it by 'allowlist'] Comma separated block number-to-hash mappings to enforce (<number>=<hash>)",
```

Comment:
We actually have a system for deprecating flags, making them not appear in the normal listing, but in the extended help. 
Also, "you shall" sounds a bit ... dramatic?